packages/cli/src/services/user.service.ts

```diff
@@ -21,10 +21,15 @@ export class UserService {
 		private readonly logger: Logger,
 		private readonly userRepository: UserRepository,
 		private readonly jwtService: JwtService,
-		private readonly mailer: UserManagementMailer,
+		private mailer: UserManagementMailer,
 		private readonly urlService: UrlService,
+		private readonly internalHooks: InternalHooks,
```

Comment:
we can't do this yet. there is still an annoying circular dependency that'll prevent the app from actually starting up.